packages/gui/src/components/settings/SettingsIntegration.tsx

```diff
@@ -82,6 +85,38 @@ export default function SettingsIntegration() {
           </SettingsText>
         </Grid>
       </Grid>
+
+      <Grid item xs={12} sm={12} lg={12}>
+        <SettingsHR />
+      </Grid>
+
+      <Grid container gap={2}>
+        <Grid item style={{ width: '624px' }}>
+          <Flex flexDirection="row" alignItems="center" justifyContent="spaceBetween" gap={1}>
+            <SettingsTitle>
+              <Trans>Reset Bypass</Trans>
+            </SettingsTitle>
+            <Flex flexDirection="row" flexGrow={1} justifyContent="flex-end" gap={2}>
+              <Button
+                onClick={() => resetBypass()}
+                color="secondary"
+                variant="outlined"
+                data-testid="SettingsPanel-resync-wallet-db"
+              >
+                <Trans>Reset</Trans>
+              </Button>
+            </Flex>
+          </Flex>
+        </Grid>
+        <Grid item container style={{ width: '400px' }} gap={2}>
+          <SettingsText>
+            <Trans>
+              Take control of your WalletConnect security with reset bypass. Click the button to revoke previous command
+              approvals and ensure all future commands require your approval.
```

Comment:
```suggestion
This will reset all previously granted permissions across all Dapps that have been connected to. After
              resetting you will be asked to grant permission again.
```